src/components/Registration/Registration.tsx

```diff
@@ -81,15 +82,9 @@ const Registration: React.FC = () => {
     }, [error])
 
     useEffect(() => {
-        switch (role) {
-            case Role.ADMIN:
-                return history.push('/menu/orders')
-            case Role.USER:
-                return history.push(`/myOffice/${id}`)
-            case Role.MASTER:
-                return history.push(`/MyWorkplace/${id}`)
-        }
-
+        const myRole = new RolesUrls(id)
```

Comment:
Do not use RolesUrls Class.
Define simple key_value
const urlByRole = {
            [Role.USER]: `/myOffice/${id}`,
            [Role.MASTER]: `/MyWorkplace/${id}`,
            [Role.ADMIN]: `/menu/orders`,
}
and get value by role, like:

const url = urlByRole[Role.MASTER]